org.servo.Servo.yaml

```diff
@@ -0,0 +1,31 @@
+app-id: org.servo.Servo
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: servo
+finish-args:
+  - --socket=x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+modules:
+  - name: servo
+    buildsystem: simple
+    build-commands:
+       - mkdir -p /app/bin/resources/
+       - cp -R resources/* /app/bin/resources/
+       - install -D resources/servo.svg /app/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+      # servo_1024.png, servo_64.png, servo.ico
+       - install -D servo /app/bin/servo
+       - install -D org.servo.Servo.desktop /app/share/applications/${FLATPAK_ID}.desktop
+       - install -D org.servo.Servo.metainfo.xml /app/share/metainfo/${FLATPAK_ID}.metainfo.xml
+    sources:
+      - type: archive
+        url: https://download.servo.org/nightly/linux/servo-latest.tar.gz
```

Comment:
Can't we build it here instead of using the binary blob directly?

Reply:
Being attempted here https://github.com/servo/servo/issues/27942#issuecomment-765905405

Reply:
It builds fine from the confines of Thunderbird, except when you only have 4GB of RAM (not a problem on the building infra)

This would also allow `aarch64` support, which Mozilla infra doesn't do:
```
% file servo/servo
servo/servo: ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 3.2.0, BuildID[sha1]=5161d2c948c592b019a98261171c40c140254830, with debug_info, not stripped
```

Do the Flathub builders really let the build succeed on aarch64? It seems that it doesn't care but the Flatpak surely will never run.

Also by using this URL you make in non-reproducible. If I try to build the Flatpak right now it will tell me it is the wrong sha.

Reply:
Sorry, meant to close this PR in favor of the https://github.com/servo/servo/pull/28088